src/vivarium_cluster_tools/psimulate/branches.py

```diff
@@ -34,13 +34,17 @@ def from_branch_configuration(
         branch_configuration_file
             Absolute path to the branch configuration file.
         """
-        branches, input_draw_count, random_seed_count = load_branch_configuration(
-            branch_configuration_file
-        )
+        (
+            branches,
+            input_draw_count,
+            random_seed_count,
+            input_draws,
+        ) = load_branch_configuration(branch_configuration_file)
         keyspace = calculate_keyspace(branches)
-        keyspace["input_draw"] = calculate_input_draws(input_draw_count)
+        keyspace["input_draw"] = (
+            input_draws if input_draws else calculate_input_draws(input_draw_count)
+        )
```

Comment:
So keyspace["input_draw"] will always be a list - either a provided one or a"randomly" chosen one on the back end?